.prettierignore

```diff
@@ -3,6 +3,7 @@ native/ios/**/*
 native/vendor/**/*
 
 native/assets/licenses_native.json
+web/assets/licenses.json
```

Comment:
Please be consistent here. Either use `licenses_native.json` and `licenses_web.json` or `licenses.json` in both cases. I'd prefer `licenses.json` since the web/native suffix is really redundant.

Reply:
done